src/template.js

```diff
@@ -710,80 +710,254 @@ module.exports = {
     meta: {
         questionnaireDocumentVersion: '4.2.0',
         onComplete: {
-            tasks: {
-                sendEmail: {
+            actions: [
+                {
+                    description: 'Confirmation email - applicant:adult',
                     type: 'sendEmail',
-                    l10n: {
-                        vars: {
-                            lng: 'en',
-                            context: {
-                                $data:
-                                    '/answers/p-applicant-who-are-you-applying-for/q-applicant-who-are-you-applying-for'
-                            },
-                            ns: 'notification-confirmation'
+                    // prettier-ignore
+                    cond: ['==', '$.answers.p-applicant-confirmation-method.q-applicant-confirmation-method', 'email'],
```

Comment:
I can see why this just 1 condition, but would it be worth being explicit and have a condition testing for not being a `proxy`?

Reply:
Probably a good shout. Nothing wrong with being more explicit. Given this has been through testing, you ok if we do this at a later date?